inventory/src/js/itemIndex.js

```diff
@@ -98,10 +98,13 @@ const ItemIndex = () => {
 
       <Stack
         direction='column'
-        justifyContent='center'
+        justifyContent='space-between'
         alignItems='center'
         spacing={3}
-        sx={{ marginTop: 1 }}
+        sx={{
+          marginTop: 1,
+          minHeight: 0.8,
```

Comment:
```suggestion
          minHeight: '80%',
```
`0.8` works too but prefer percentages, seems clearer